signing_service/src/constants.py

```diff
@@ -0,0 +1,3 @@
+SIGNING_SERVICE_DEFAULT_PORT = 8888
```

Comment:
Middleman uses 8888 for accepting connections from Concent (the internal port). You should use the external port there.

Reply:
I have answered in https://github.com/golemfactory/concent/pull/619#discussion_r202466505

Reply:
:+1: done